src/dev/ci_setup/setup.sh

```diff
@@ -160,6 +160,12 @@ if [ "$GIT_CHANGES" ]; then
   exit 1
 fi
 
+###
+### Elastic APM Node.js Agent configuration
+###
+ELASTIC_APM_ACTIVE=true
+ELASTIC_APM_ENVIRONMENT=ci
```

Comment:
Will need to export this so they are available to the Kibana process.


```suggestion
export ELASTIC_APM_ACTIVE=true
export ELASTIC_APM_ENVIRONMENT=ci
```

Reply:
fixed